app/controllers/case_contacts_controller.rb

```diff
@@ -1,5 +1,6 @@
 # CaseContactsController with default actions
 class CaseContactsController < ApplicationController
+  before_action :authenticate_user!
```

Comment:
Oooooops
We need some "everything" security tests- i.e. iterate over all endpoints and test that they all require auth (except login). I made a ticket: https://github.com/rubyforgood/casa/issues/207